src/Microsoft.HttpRepl.Telemetry/ITelemetry.cs

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+
+namespace Microsoft.HttpRepl.Telemetry
+{
+    public interface ITelemetry
+    {
+        bool Enabled { get; }
```

Comment:
nit: `IsEnabled` - I think we usually prefer the `Is` prefix on state properties.